### PR TITLE
Steveholmgren.drlc server.1

### PR DESCRIPTION
**Pull request by steveholmgren**
> !!!!!!!!!! Please delete the instructions below and replace with PR description
>
> If you have an issue number, please use a syntax of
> `Fixes #12345` and a brief change description
>
> If you do not have an issue number, please have a good description of
> the problem and the fix. Help the reviewer understand what to expect.
>
> Make sure you delete these instructions (to prove you have read them).
>
> !!!!!!!!!! Instructions end

### CODE REVIEW

**Review by andy31415 — CHANGES_REQUESTED** [review]
@steveholmgren 

please update title and description of this PR.

File: src/app/clusters/drlc-server/Commands.h [hunk 1]
```diff
@@ -0,0 +1,1681 @@
...
+    }
+
+    CHIP_ERROR SendCommand(MTRBaseDevice * device, chip::EndpointId endpointId) override
+    {
+        constexpr chip::ClusterId clusterId = chip::app::Clusters::DemandResponseLoadControl::Id;
+        constexpr chip::CommandId commandId = chip::app::Clusters::DemandResponseLoadControl::Commands::AddLoadControlEventRequest::Id;
+
+        ChipLogProgress(chipTool, "Sending cluster (0x%08" PRIX32 ") command (0x%08" PRIX32 ") on endpoint %u", clusterId, commandId, endpointId);
+
+        dispatch_queue_t callbackQueue = dispatch_queue_create("com.chip.command", DISPATCH_QUEUE_SERIAL);
```
Comment (andy31415) [inline src/app/clusters/drlc-server/Commands.h:179]: The placement of these files does not look right: there should be no darwin-specific code in src/app/clusters nor should there be generated code in src/app

File: src/app/clusters/drlc-server/app-common/zap-generated/Attributes.h
Comment (andy31415) [inline src/app/clusters/drlc-server/app-common/zap-generated/Attributes.h:1]: These generated files should not exist.

**Comment by steveholmgren** [comment]
Closing - it was made from the  UI my mistake

**Pull request closed without merging**